static/js/bundles/app.js

```diff
@@ -207,6 +207,8 @@ import "spectrum-colorpicker/spectrum.css";
 import "katex/dist/katex.css";
 import "flatpickr/dist/flatpickr.css";
 import "flatpickr/dist/plugins/confirmDate/confirmDate.css";
+import "../../styles/themes/default.scss";
+import "../../styles/themes/night_mode.scss";
```

Comment:
Can you explain the thinking behind the reordering here?

Reply:
@andersk FYI ; I don't recall whether the ordering is important in this file for other reasons.

Reply:
`night_mode.css` can be put anywhere in this list since it has a unique scope (`body.night-mode`). I thought it would be neat to group themes together.